src/components/contextual/pages/vebal/MyVeBAL/components/MyVeBalCards.vue

```diff
@@ -104,7 +104,7 @@ const cards = computed(() => {
       id: 'myLockedLpToken',
       label: t('veBAL.myVeBAL.cards.myLockedLpToken.label'),
       value: isWalletReady.value
-        ? fNum2(props.lockedFiatTotal, FNumFormats.fiat)
+        ? fNum2(props.totalLockedValue, FNumFormats.fiat)
```

Comment:
Out of scope of this PR: 
Why not having a less verbose helper function: 
`toFiat(props.totalLockedValue)`
all across the app. (Same with other popular `fNum2` usages.)

Reply:
We should but we already have a `toFiat` function in `useNumbers` that would need to be refactored. It currently takes a token amount and address and tries to return a fiat amount. I think we should rename that function `tokenToFiatLabel` and add a new function `toFiatLabel` that does what you suggest, turn a fiat value into a formatted fiat string.

Reply:
Ok, I just added helper functions here: https://github.com/balancer-labs/frontend-v2/commit/2b294ae45ac8b251c4ce2d70f84633c97700e39f

We should do a search and replace in another PR.